### PR TITLE
fix(meta): display distinct dedup tables in dashboard

### PR DESCRIPTION
**Pull request by richardchien**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

As title.

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
- [x] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

**Review by st1page — APPROVED** [review]
LGTM

File: src/meta/src/stream/stream_graph/visit.rs [hunk 1]
```diff
@@ -93,6 +93,9 @@ where
                         always!(s.table, "HashAgg");
                     }
                 }
+                for (distinct_col, dedup_table) in &mut node.distinct_dedup_tables {
```
Comment (BugenZhao) [inline src/meta/src/stream/stream_graph/visit.rs:96]: How could it work without assigning a global table ID before this PR? 🥵

Comment (richardchien) [reply src/meta/src/stream/stream_graph/visit.rs:96]: I think table id is assigned in this PR😄

Comment (BugenZhao) [reply src/meta/src/stream/stream_graph/visit.rs:96]: If the table ID is local for each fragment graph (mview), then tables from multiple views may get conflicted. 😄 I guess it'll be better to assign a range, for example, from 1000, for global table IDs, then we can check it out earlier. Not a big deal.

**Review by BugenZhao — APPROVED** [review]

**Pull request merged**